examples/mysql-wordpress-pd/README.md

```diff
@@ -34,301 +34,231 @@ Documentation for other releases can be found at
+This example describes how to run a persistent installation of
+[WordPress](https://wordpress.org/) on Kubernetes, using
+[Persistent Volumes](../../docs/user-guide/persistent-volumes.md) and
+[Secrets](../../docs/user-guide/secrets.md).
+
+We'll use the [mysql](https://registry.hub.docker.com/_/mysql/) and
+[wordpress](https://registry.hub.docker.com/_/wordpress/) official
+[Docker](https://www.docker.com/) images for this installation. (The
+wordpress image includes an Apache server). We'll create two Kubernetes
+[Deployments](../../docs/user-guide/deployments.md) to run MySQL and
+WordPress, both with associated persistent volmes and
+[Services](../../docs/user-guide/services.md) to front each
+deployment.
+
+This example demonstrates several useful things, including: how to set
+up and use persistent disks with Kubernetes pods; how to define
+Kubernetes services to enable pods to find each other; and use of an
+external load balancer to expose the WordPress service externally and
+make it transparent to the user if the WordPress pod moves to a
+different cluster node.
+
+## tl;dr Quickstart
+
+Put your desired mysql password in a file called `password.txt` with no
+trailing newline.
+
+```shell
+kubectl create -f https://raw.githubusercontent.com/kubernetes/kubernetes/master/examples/mysql-wordpress-pd/local-volumes.yaml
+kubectl create secret generic mysql-pass --from-file=password.txt
+kubectl create -f https://raw.githubusercontent.com/kubernetes/kubernetes/master/examples/mysql-wordpress-pd/mysql-deployment.yaml
+kubectl create -f https://raw.githubusercontent.com/kubernetes/kubernetes/master/examples/mysql-wordpress-pd/wordpress-deployment.yaml
 ```
 
-Next, start up a Kubernetes cluster:
-
-```sh
-wget -q -O - https://get.k8s.io | bash
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+##Table of Contents
+
+  - [Cluster Requirements](#cluster-requirements)
+  - [Decide where you will store your data](#decide-where-you-will-store-your-data)
+    - [GCE Persistent Disk](#gce-persistent-disk)
+    - [Host Path](#host-path)
+  - [Create the MySQL Password Secret](#create-the-mysql-password-secret)
+  - [Deploy MySQL](#deploy-mysql)
+  - [Deploy WordPress](#deploy-wordpress)
+  - [Visit your new WordPress blog](#visit-your-new-wordpress-blog)
+  - [Take down and restart your blog](#take-down-and-restart-your-blog)
+  - [Next Steps](#next-steps)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Cluster Requirements
+
+Consult a
+[Getting Started Guide](../../docs/getting-started-guides/)
+to set up a cluster and the kubectl command line client.
+
+Kubernetes runs in a variety of environments and is inherently
+modular. Not all clusters are the same. These are the requirements for
+this example.
+
+* Kubernetes version 1.2 is required due to using newer features, such
```

Comment:
Link to 1.2? And/or a document that explains how to check?

http://releases.k8s.io
https://github.com/kubernetes/kubernetes/blob/master/docs/user-guide/kubectl/kubectl_version.md